source/renderer/app/components/voting/voting-registration-wizard-steps/VotingRegistrationStepsEnterPinCode.js

```diff
@@ -122,6 +138,21 @@ export default class VotingRegistrationStepsEnterPinCode extends Component<Props
     }
   );
 
+  pinFocusValidity = (
```

Comment:
Isn't a PIN only valid if you entered all 4 digits? This function suggests that any PIN between 1-4 digits would be valid. Also the function name `pinFocusValidity` is confusing – something like `arePinCodesValid` or since this function actually only checks the length of the PIN codes: `arePinCodesFullyEntered`

Reply:
@DominikGuzei yes it's valid in the case when you have entered all 4 digits and repeated them in the same order. But this function just makes sure that you have at least 1 value entered or not in the left or right area of pin codes component so it can determine where is your focus currently. Point is to just determine if it has value and where that value is. 

So i would suggest naming this to something like: checkPinCodesFieldFocus

Reply:
@aleksandardjordjeviciohk and me discussed that this method can safely be removed